src/organisms/actionBar/actionBar.stories.tsx

```diff
@@ -27,15 +28,15 @@ export default {
 const Template: ComponentStory<typeof ActionBar> = (args) => {
   const { chipFilters: filters } = args;
   const [chipValue, setChip] = useState<number>();
-  const [selectValue, setSelectedValue] = useState<SelectGroupOption>();
+  const [selectValue, setSelectedValue] = useState<SelectOption[]>([]);
   return (
-    <ActionBar<'group'>
+    <ActionBar<'multi-group'>
       chipFilterValue={chipValue as number}
```

Comment:
can you investigate the casting here? Might give the impression of broken component if something stops working